### ISSUE TITLE
a11y: Unable to use keyboard

### ISSUE THREAD

**Issue by franck-paul** [nhn/tui.editor#1132]
The toolbar cannot be reached from the keyboard, nor can its buttons be navigated with the keyboard arrows.

As tested on https://nhn.github.io/tui.editor/latest/tutorial-example01-editor-basic page.

**Comment by Luwangel** [comment]
We also need this kind of feature to use this editor in the enterprise edition of React-Admin (https://github.com/marmelab/react-admin).

Do you plan to work on it soon?

**Comment by Ats1999** [comment]
I need help in implementing this markdown editor.

**Comment by seonim-ryu** [comment]
@franck-paul I have a question. As you said, to apply a11y to toolbar buttons, I think it should be a UX that uses the tab index, not the arrow keys. Is it what I understood? If not, is there a model you are referring to?

**Comment by franck-paul** [comment]
Nope, I really talk about arrows key, as for example, it is possible with the toolbar available on my blog when someone comment one of my post (https://open-time.net/post/2020/08/06/Master-to-main) ; when the focus is inside the textarea you can fired a shift-tab to focus in the toolbar and then select a button with arrow keys.

**Comment by seonim-ryu** [comment]
@franck-paul You seem to be talking about the behavior of this editor. Right?
![a11y](https://user-images.githubusercontent.com/18183560/89626698-bb951f00-d8d4-11ea-94bb-2f15227f5d3c.gif)

If it is right, I think a11y function should be provided as an option in order not to break the current UI of the Editor. It is difficult to deal with it right now because the design has to be considered together, but I will try to apply the function as much as possible.

**Comment by franck-paul** [comment]
Yes absolutely, it is this feature and imho the a11y (using only the keyboard) should not be an option. Anyway I'm happy that you will take care, as far as possible, about this enhancement.

Have you tried using your editor only using keyboard (whithout shortkeys)?

**Comment by seonim-ryu** [comment]
@franck-paul Sorry for the late answer.

> Have you tried using your editor only using keyboard (whithout shortkeys)?

In your question, you asked me if I used the Editor only with keyboard without shortkeys (I think this is a shortcut). But I don't understand what this means. Can you explain in more detail?

**Comment by franck-paul** [comment]
Yes, sorry I thought "shortcut" and typed "shortkey" :-)

It appears that is not easy to learn the "shortcut" of all of the different buttons of the toolbar, if there is a shortcut for each of them! I just tried to search them in doc but without finding them. They (the shortcuts) also not displayed on mouse-over on button.

So how can I make a word strong (bold) in these conditions?
